OpenRA.Game/Traits/DrawLineToTarget.cs

```diff
@@ -72,6 +72,9 @@ public IEnumerable<IRenderable> RenderAfterWorld(WorldRenderer wr)
 				if (target.Type == TargetType.Invalid)
 					continue;
 
+				if (target.IsVisibleFor(self))
```

Comment:
The boolean logic has been swapped here. This should be `if (!target.IsVisibleFor(self))` instead.